pkg/restapi/operation/didexchange/didexchange_test.go

```diff
@@ -455,26 +454,6 @@ func TestServiceEvents(t *testing.T) {
 	}
 }
 
-func TestOperationEventError(t *testing.T) {
-	client, err := didexchange.New(&mockprovider.Provider{StorageProviderValue: mockstore.NewMockStoreProvider(),
-		ServiceValue: &protocol.MockDIDExchangeSvc{}})
-	require.NoError(t, err)
-
-	ops := &Operation{client: client, actionCh: make(chan service.DIDCommAction)}
-
-	aCh := make(chan service.DIDCommAction)
-	err = client.RegisterActionEvent(aCh)
-	require.NoError(t, err)
-
-	err = ops.startClientEventListener()
-	require.Error(t, err)
-	require.Contains(t, err.Error(), "didexchange action event registration failed: channel is already "+
```

Comment:
Why is this test case removed ? I don't see any API changes in client pkg.

Reply:
We are using `ServiceValue: &protocol.MockDIDExchangeSvc{}` it mocks our `RegisterActionEvent` and it is always without error.  This test does not make sense.

Reply:
reverted it